### DISCUSSION TITLE
How to enable CONFIG_SPIRAM_SUPPORT in Arduino

### DISCUSSION THREAD

**Discussion by tanmanh0707** [Bodmer/TFT_eSPI#2800] [Q&A - General]
Hi Bodmer,

I'm using esp32-s3 n8r2 which has 2MB PSRAM and I want to use it for the tft display.
I read the source code and there is a macro like this:
```
#if defined (ESP32) && defined (CONFIG_SPIRAM_SUPPORT)
```
CONFIG_SPIRAM_SUPPORT is a configuration in Kconfig if esp-idf is used. However, I'm using Arduino platform so I cannot enable this configuration.
Could you please let me know how to enable this CONFIG_SPIRAM_SUPPORT option using Arduino?
And only this option is enabled will force the library uses PSRAM using ps_calloc, isn't it?
I'm sure my board has PSRAM with psramInit and psramFound returns true.

Thank you very much,
Manh

**Comment by Bodmer** [comment]
If the SPIRAM is enabled in the Arduino IDE then the library will automatically use SPIRAM for sprites.

**Reply by tanmanh0707** [reply]
Dear Bodmer,

As I read somewhere, in latest arduino esp32 version, if PSRAM is enabled. it automatically uses PSRAM in every malloc, calloc without calling ps_malloc, ps_calloc. Is this what you mean?

Best regards,
Manh
